docs/errors/adding-custom-dto.md

```diff
@@ -0,0 +1,41 @@
+---
+id: adding-custom-dto-to-app
+title: Adding custom DTOs to a generated app
+tags:
+ - Prisma
+ - GraphQL
+ - DTO
+
+sidebar_label: Adding custom DTOs to a generated app
```

Comment:
This article is about adding DTO, and the asset you added is about DB error when a process is already running on ports 5432. These are two different things.
Regarding this file, I think we should provide a more specific example with our sample app

Reply:
@yuval-hazaz Do you want me to take a sample app, add a property with Enum type (options) to one of the entities, and add filter enum input as a custom code? Then write a guide on how I made it, and Moshe will add it to the docs?